src/cdk/v2/destinations/lytics/procWorkflow.yaml

```diff
@@ -28,7 +28,8 @@ steps:
         ...flattenTraits,
```

Comment:
move out fields mapping to json config file

Reply:
done